### PR TITLE
Preparation for v2023.2 release

### PR DESCRIPTION
**Pull request by tgreenx**
## Purpose

Preparation for v2023.2 release

## Changes

- Supported versions in prerequisites document

## How to test this PR

N/A. Review only.

### CODE REVIEW

**Review by matsduf — DISMISSED** [review]

File: docs/public/installation/prerequisites.md [hunk 1]
```diff
@@ -35,15 +35,14 @@ at large cloud providers.
...
-Rocky Linux 9.2  | 10.5    | 13.10
-Ubuntu 20.04     | 10.3   | 12.15
-Ubuntu 22.04     | 10.6   | 14.8
+FreeBSD 14.0     | 10.11   | 16.1
+Rocky Linux 8.9  | 10.5    | 15.3
+Rocky Linux 9.3  | 10.5    | 15.3
+Ubuntu 20.04     | 10.3    | 12.17
+Ubuntu 22.04     | 10.6    | 14.10
 
-* (*) FreeBSD uses MySQL, not MariaDB.
```
[the remark this answers is not in the document]
Comment (marc-vanderwal) [reply docs/public/installation/prerequisites.md:46]: Based on https://cgit.freebsd.org/ports/tree/Mk/bsd.default-versions.mk, by default, packages are linked against libraries for MySQL 8.0. This applies to all supported versions of FreeBSD.

For PostgreSQL, the default major version is 15.x.

**Review by pnax — DISMISSED** [review]
Looks good for Ubuntu 22.04

File: docs/public/installation/prerequisites.md [hunk 2]
```diff
@@ -35,13 +35,13 @@ at large cloud providers.
...
-Debian 11        | 10.5    | 13.10
+Debian 12        | 10.11   | 15.5
 Docker           | n/a     | n/a
-FreeBSD 13.2     | 8.0 (*) | 13.11
-Rocky Linux 8.8  | 10.3    | 10.23
-Rocky Linux 9.2  | 10.5    | 13.10
-Ubuntu 20.04     | 10.3   | 12.15
-Ubuntu 22.04     | 10.6   | 14.8
+FreeBSD 14.0     | 8.0 (*) | 15
+Rocky Linux 8.9  | ?       | ?
```
Comment (mattias-p) [inline docs/public/installation/prerequisites.md:41]: * MariaDB 10.3
* PostgreSQL 10.23

File: docs/public/installation/prerequisites.md [hunk 3]
```diff
@@ -56,11 +56,11 @@ Ubuntu 22.04     | 10.6   | 14.8
...
 Operating System | Perl
 ---------------- | ----
-Debian 11        | 5.32
+Debian 12        | 5.36
 Docker           | (*)
-FreeBSD 13.1     | 5.32
-Rocky Linux 8.8  | 5.26
-Rocky Linux 9.2  | 5.32
+FreeBSD 14.0     | 5.38
+Rocky Linux 8.9  | 5.32
```
Comment (mattias-p) [inline docs/public/installation/prerequisites.md:62]:
```suggestion
Rocky Linux 8.9  | 5.26
```

**Review by marc-vanderwal — APPROVED** [review]

File: docs/public/installation/prerequisites.md [hunk 4]
```diff
@@ -56,11 +56,11 @@ Ubuntu 22.04     | 10.6   | 14.8
 
 Operating System | Perl
 ---------------- | ----
-Debian 11        | 5.32
+Debian 12        | 5.36
 Docker           | (*)
-FreeBSD 13.1     | 5.32
-Rocky Linux 8.8  | 5.26
-Rocky Linux 9.2  | 5.32
+FreeBSD 14.0     | 5.38
```
Comment (matsduf) [inline docs/public/installation/prerequisites.md:61]: It should be 5.36.

**Review by marc-vanderwal — APPROVED** [review]

**Review by matsduf — APPROVED** [review]

**Comment by matsduf** [comment]
Can this be merged now?

**Pull request merged**